netci.groovy

```diff
@@ -0,0 +1,19 @@
+import jobs.generation.Utilities
+
+def project = GithubProject
+def branch = GithubBranchName
+def isPr = true
+def containerOs = 'Debian' 
+def cliChannel = 'rel-1.0.0'
+def newJobName = Utilities.getFullJobName(project, containerOs, isPr)
+
+def newJob = job(newJobName) {
+    steps {
+        shell("./test/build-images.sh ${cliChannel}")
+        shell("./test/run-test.sh ${cliChannel}")
```

Comment:
Good point.  I've also changed it to use Ubuntu 16.04 machines with the `latest-or-auto-docker` label.